packages/language-server/src/plugins/PluginHost.ts

```diff
@@ -420,11 +438,18 @@ export class PluginHost implements LSProvider, OnWatchFileChanges {
         args: any[],
         mode: ExecuteMode.Collect
     ): Promise<T[]>;
+    private execute<T>(
```

Comment:
We don't need this additional type definition and the `if` inside `ExecuteMode.Collect` anymore, right?